lib/puppet/parser/functions/npm_version.rb

```diff
@@ -0,0 +1,11 @@
+module Puppet::Parser::Functions
+  newfunction(:npm_version, :type => :rvalue) do |args|
+    node_unpack_folder = args.shift
+    version            = `#{node_unpack_folder}/bin/npm -v`
```

Comment:
should be wrapped into a function as executing this command causes issues with the tests